templates/NamedOperationGroupFile.stencil

```diff
@@ -23,12 +23,11 @@ public final class {{ group.name }} {
         self.commonOptions = client.commonOptions
     }
 
-    public func url(forTemplate templateIn: String, withKwargs kwargs: [String: String]? = nil) -> URL? {
-        return client.url(forTemplate: templateIn, withKwargs: kwargs)
+    public func url(forTemplate templateIn: String, withKwargs kwargs: [String: String]? = nil, and addedParams: [QueryParameter]? = nil) -> URL? {
+        return client.url(forTemplate: templateIn, withKwargs: kwargs, and: addedParams)
```

Comment:
What's the case for this?

Reply:
I create a helper method so that it builts an URL based on urltemplate and query parameters. This is different from AzureCore , which has 1 method to build url and another to add query parameters.